code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Other.dm

```diff
@@ -322,6 +322,23 @@
 	for (var/obj/effect/decal/cleanable/blood/B in T)
 		qdel(B)
 
+/datum/reagent/lube
+	name = "Slippery Motor Oil"
```

Comment:
```suggestion
	name = "slippery motor oil"
```

Reply:
All reagent names are written in upper case.